tools/debian/rules

```diff
@@ -12,6 +12,10 @@ endif
 %:
 	dh $@
 
+clean:
+	rm -f setup.py  # otherwise dh_clean thinks we're a python package
+	dh clean
```

Comment:
If the `--buildsystem` above does not work, please do this as
```
execute_before_dh_clean:
        rm -f setup.py
```
but this would still be rather nasty, as it will at least cause a warning during source package build that an original file got removed.

Reply:
I also agree that this would be better than what I did.

Reply:
Not doing this one, since the above idea worked.